extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -23,5 +23,12 @@
 		border: 0;
 		height: 1px;
 	}
-}
 
+	&__counter {
+		color: $color-text;
+		float: right;
```

Comment:
I wonder if we could use flex and have two containers in page header - one for title/subtitle/categories and second for buttons/counters/languages - similar like we have right now in page header experiment.